example-junit/src/test/java/com/epam/reportportal/example/junit/extension/RetryTest.java

```diff
@@ -1,22 +1,39 @@
 package com.epam.reportportal.example.junit.extension;
 
 import org.junit.Assert;
+import org.junit.BeforeClass;
 import org.junit.Ignore;
 import org.junit.Test;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import com.nordstrom.automation.junit.JUnitConfig.JUnitSettings;
+
+import static org.junit.Assert.assertTrue;
+
 import java.util.concurrent.atomic.AtomicInteger;
 
 @Ignore
 public class RetryTest {
 
+	private static int count;
 	private static final Logger LOGGER = LoggerFactory.getLogger(RetryTest.class);
 
 	private static final int RETRY_NUMBER = 20;
 	private static final AtomicInteger COUNTER = new AtomicInteger(0);
-
-//	@Ignore
+	
+	@BeforeClass
+	public static void beforeClass() {
+		System.setProperty(JUnitSettings.MAX_RETRY.key(), "3");
```

Comment:
The automatic retry feature of `JUnit Foundation` is activated by setting the `junit.max.retry` System property to a positive integer value. This example isn't the typical way to specify this, however. Generally, you'd specify the maximum retry attempts in the `junit.properties` configuration file or on the command line.